Database/Corrections/cataQuestFixes.lua

```diff
@@ -1142,6 +1145,7 @@ function CataQuestFixes.Load()
             [questKeys.nextQuestInChain] = 26465,
         },
         [13621] = { -- Gorat's Vengeance
+            [questKeys.extraObjectives] = {{nil,Questie.ICON_TYPE_INTERACT,l10n("Use the imbued blood"),0,{{"monster",33294}}}},
```

Comment:
If you add any new string that is used in `l10n(...)` then that also needs to be added to this list:

https://github.com/Questie/Questie/blob/master/Localization/Translations/ExtraObjectives/CataObjectives.lua